### PR TITLE
Make file preview layout less dynamic, to prevent layout loop

### PR DESCRIPTION
**Pull request by ManWithBear**
Looks like our layout was too dynamic and layout engine goes mad with it. Specifically:
- `ChatFileAttachmentListView.ItemView` 2 components that limit min height, positioned in the middle, nothing to limit max size
- `ChatMessageContentView.messageReactionsView` in general bad constraints. Will be reworked with reactions tail update

### CODE REVIEW

File: Sources_v3/StreamChatUI/ChatChannel/ChatMessage/ChatMessageContentView.swift [hunk 1]
```diff
@@ -205,9 +205,9 @@ open class ChatMessageContentView<ExtraData: ExtraDataTypes>: View, UIConfigProv
         }
 
         if message?.deletedAt == nil, !(message?.reactionScores.isEmpty ?? true) {
-            toActivate.append(bubbleToReactionsConstraint!)
+//            toActivate.append(bubbleToReactionsConstraint!)
         } else {
-            toDeactivate.append(bubbleToReactionsConstraint!)
+//            toDeactivate.append(bubbleToReactionsConstraint!)
```
Comment (ManWithBear) [inline Sources_v3/StreamChatUI/ChatChannel/ChatMessage/ChatMessageContentView.swift:210]: Reaction view will be back when reaction tail is finished

File: StreamChat_v3.xcodeproj/xcshareddata/xcschemes/DemoApp.xcscheme [hunk 2]
```diff
@@ -50,6 +50,12 @@
             ReferencedContainer = "container:StreamChat_v3.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-UIViewLayoutFeedbackLoopDebuggingThreshold 100"
```
Comment (ManWithBear) [inline StreamChat_v3.xcodeproj/xcshareddata/xcschemes/DemoApp.xcscheme:55]: I think we should keep this flag for future loop cases

Comment (VojtaStavik) [reply StreamChat_v3.xcodeproj/xcshareddata/xcschemes/DemoApp.xcscheme:55]: 👍

**Review by VojtaStavik — APPROVED** [review]
Thanks for fixing this @ManWithBear 👍

**Pull request merged**